specifications_dbus.md

```diff
@@ -0,0 +1,92 @@
+# Specifications
+
+## Index
+
+* [Connection](#connection)
+* [Connector](#connector)
+* [Connector API](#connector-API)
+* [Distributor](#distributor)
+* [Distributor API](#distributor-api)
+* [Appendix: Implementation practices](#appendix-implementation-practices)
+
+## Connection
+
+A *connection* is created when a connector registers with a distributor. It is uniquely identified by its *token*, a random string which the connector generates as part of the registration process.
```

Comment:
Definitions should be in definitions.md file. If you feel it misses some, like connection, can you add it to this file?